plug_gateway/lib/plug_gateway/router.ex

```diff
@@ -1,13 +1,37 @@
 defmodule PlugGateway.Router do
+  @moduledoc """
+  PlugGateway router.
+
+  Should you copy from _this_ code?
+
+  * Copy the use of `Plug.Telemetry` if you want `OpentelemetryPlug` to work. That aside, there's
+    nothing in this module particularly relevant to tracing or metrics.
+
+  * Copy the config pattern if you want `Vapor`'s `.env` and config file support, error messages
+    _etc._ and don't mind the overhead. Otherwise, a `System.get_env/2` call in your
+    `config/config.exs` and an `Application.get_env/2` call here will suffice.
+
+  * Copy `use Norm` ... `@contract` if you prioritise rapid bug isolation over compact code.
+  """
+
   use Plug.Router
+  use Norm
+
+  defdelegate config, to: PlugGateway.Config
+  defdelegate get(url), to: PlugGateway.BackendClient
 
-  alias PlugGateway.BackendClient
+  # This specific event_prefix required for opentelemetry_plug at e750fd6 on update-name:
+  plug(Plug.Telemetry, event_prefix: [:plug_adapter, :call])
```

Comment:
Doesn't `PlugCowboy` emit the event without the user needing to do it themselves?

Reply:
It does _now_; I need to upgrade it in this directory.